graphql_compiler/compiler/expressions.py

```diff
@@ -928,11 +937,22 @@ def to_sql(self, dialect: Any, aliases: AliasesDictType, current_alias: AliasTyp
             # PostgreSQL
             # coalesce to an empty array of the corresponding type
             inner_type = strip_non_null_from_type(self.field_type.of_type)
-            if is_same_type(GraphQLInt, inner_type):
-                sql_array_type = "INT"
-            elif is_same_type(GraphQLString, inner_type):
-                sql_array_type = "VARCHAR"
-            else:
+            graphql_type_to_sql_array_type_dict = {
+                GraphQLInt: "INT",
```

Comment:
Are we confident `INT` and not `BIGINT` or something else like it is the appropriate choice here?

Reply:
FYI `BIGINT` returns as a string on db-fiddle.

https://www.db-fiddle.com/f/fJtQSrZqZn8tqd1S3ZfQWF/0

Seems like it might be a javascript limitation in db fiddle? ex: https://stackoverflow.com/questions/39168501/pg-promise-returns-integers-as-strings

Reply:
Fascinating! It's probably exactly the Javascript issue you found, and doesn't seem to be a concern in Python:
> PostgreSQL bigint and oid are converted to long in Python 2 and to int in Python 3.
https://www.postgresql.org/docs/current/plpython-data.html#id-1.8.11.11.3